study.md

```diff
@@ -43,7 +46,8 @@ Although both Array and Enumerable have a `count` method, Array also defines the
 method `length`.  Why is `length` sensibly defined on Array but not Enumerable?
 
 ```md
-<!-- your answer here -->
+if there was an array with 2 indexes it would have 1 more that that array's count if it were done with length.
```

Comment:
`length` makes sense on `Array` because arrays have a finite length.  It does not make sense for an `Enumerable` to have a `length` because it might not be finite.  For example, an enumerable that returns a digit of π on each iteration does not have a finite length.